src/app/VerifyActivities/components/ActionsComponent.js

```diff
@@ -1,11 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { ButtonComponent } from '../../common/components';
 
-const ActionsComponent = () => (
+const ActionsComponent = ({ handleVerify, id }) => (
   <div className='actions'>
-    <ButtonComponent className='action--reject'><div className='cross'>&times;</div></ButtonComponent>
-    <ButtonComponent className='action--approve'><div className='checkmark' /></ButtonComponent>
+    <ButtonComponent
+      className='action--reject'
+      onClick={() => handleVerify(id, 'rejected')}
+    >
+      <div className='cross'>&times;</div>
+    </ButtonComponent>
+    <ButtonComponent
+      className='action--approve'
+      onClick={() => handleVerify(id, 'approved')}
+    >
+      <div className='checkmark' />
+    </ButtonComponent>
   </div>
 );
 
+ActionsComponent.defaultProps = {
+  handleVerify: null,
```

Comment:
This component will not be used to verify only, it would be better to rename this prop, `handleVerify`, to a name that is not limited to verifying only.

Reply:
@Chris-Maina I have fixed this.